app/serializers/api/v1/user_serializer.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Api::V1::UserSerializer
+  include FastJsonapi::ObjectSerializer
+
+  attribute :email, if: Proc.new { |record, params|
+    params[:has_email_access] == true || record.admin
+  }
+
+  attributes :name, :admin,
+             :country, :educational_institute,
+             :subscribed, :created_at, :updated_at
```

Comment:
We don't need to expose `:subscribed, :created_at, :updated_at` attributes of other users in api